pkg/kubectl/cmd/label.go

```diff
@@ -185,10 +185,12 @@ func (o *LabelOptions) RunLabel(f cmdutil.Factory, cmd *cobra.Command) error {
 		return err
 	}
 
+	includeUninitialized := cmdutil.GetFlagBool(cmd, "include-uninitialized")
```

Comment:
See my comment at  https://github.com/kubernetes/kubernetes/issues/49035#issuecomment-322322225 I think `label` works with individual resources (and doesn't work in batch mode) so it should not have this flag.